npc/custom/breeder.txt

```diff
@@ -14,20 +14,20 @@ prontera,124,201,1	script	Universal Rental NPC	726,{
 	if (ismounting()) {
 		message strcharinfo(0),"You must first remove your mount.";
 		end;
-	} else if (((eaclass()&EAJ_THIRDMASK)==EAJ_RANGER) && !countitem(6124)) {
+	} else if (((eaclass()&EAJ_THIRDMASK)==EAJ_RANGER || (eaclass()&EAJ_THIRDMASK)==EAJ_WINDHAWK) && !countitem(6124)) {
```

Comment:
The changes in this file are not necessary and invalid.
EAJ_THIRDMASK does not have the bit for fourth class and rebirth set, so this can never be true.
Also the existing check should be good enough.